server.go

```diff
@@ -95,13 +100,16 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 
 	interval, err := time.ParseDuration(conf.Interval)
 	if err != nil {
+		logrus.Fatalf("Error parsing configuration %s", err)
```

Comment:
Oversight on my part, will fix. (I'm in a lot of meetings today and sorta rushed this one, yours is def a better solution)